Sources/Graphics/shape/Polygon.swift

```diff
@@ -0,0 +1,36 @@
+import Utils
+
+public struct Polygon<T: IntExpressibleAlgebraicField> {
+    public let paths: [[Vec2<T>]]
```

Comment:
Do we want polygons to be able to represent disjoint shapes? Is there any advantage over just creating multiple polygons in that case?

Reply:
Ah okay, thanks for clarifying, that sounds reasonable!